changelogs/unreleased/gh-6833-encode-body-http-request.md

```diff
@@ -1,6 +1,7 @@
 ## feature/lua/http client
 
-* HTTP client is able to encode Lua objects automatically when it is possible.
-  Encoding depends on content type passed to HTTP request and following types
-  are supported: "application/json", "application/yaml" and
-  "application/msgpack". Now user can define his own encoding function (gh-6833).
+* Now the HTTP client is able to encode Lua objects automatically when it is possible.
```

Comment:
Nit: please, fit the lines to 80 symbols.

Reply:
Done. Thanks

Reply:
That's done, but now the entry is not a markdown list items -- it lacks the asterisk.

Reply:
Ouch, my bad. Fixed.